modules/how-to-plan-your-environment-according-to-application-requirements.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/planning-your-environment-according-to-object-limits.adoc
+
+[id='how-to-plan-according-to-application-requirements']
+= How to plan your environment according to application requirements
+
+Consider an example application environment:
+
+[options="header",cols="5"]
+|===
+|Pod Type |Pod Quantity |Max Memory |CPU Cores |Persistent Storage
+
+|apache
+|100
+|500MB
```

Comment:
Unless these are specific variables, you should have a space between the number and unit, eg "500 MB."